### PR TITLE
fix: general intro content on Initiatives page is editable via WordPress (resolves #391)

### LINKED ISSUE #391: The general text on the Initiatives page should be editable by content editors
## Is your feature request related to a problem? Please describe.

Recently, the Initiatives page has been re-implemented by fetching workshop data from Airtable. However, other than workshop information, the first introduction paragraph on Initiatives page is non-workshop information that is now hardcoded in the template code.

This paragraph, the page title and even the page url should be editable to content editors.

## Describe the solution you'd like

As discussed at the project check in meeting on Sep 14, this paragraph only contains text in the foreseeable future, which allows Airtable to be the editing tool. The solution is to create a new "initiatives_general" table in Airtable for these information: slug, title, introduction.

Note: Airtable supports text styles in markdown syntax but is not a rich editor as WordPress.

## Describe alternatives you've considered

In the future if the introduction paragraph requires richer content, a potential solution is to use WordPress to edit this paragraph and Airtable to save workshops and comments data. The code will fetch data from both data sources and assemble the webpage.

## Additional context

A screenshot of fields that need to be editable by content editors:

Screen Shot 2020-09-15 at 9 42 57 AM

### PR DESCRIPTION
**Pull request by cindyli**
* [X] This pull request has been linted by running `npm run lint` without errors
* [X] This pull request has been tested by running `npm run start` and reviewing affected routes
* [X] This pull request has been built by running `npm run build` without errors
* [X] This isn't a duplicate of an existing pull request

## Description

Aassemble "Initiatives" page by fetching the intro content from WordPress and workshop data from Airtable.

## Steps to test

1. The "Initiatives" item is available on the top navigation bar; 
2. Go to "Initiatives" page, the intro paragraph matches the "Initiatives" page defined in WP. The following workshop information matches what's defined in Airtable.

### CODE REVIEW

File: src/page.njk [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ---
 pagination:
-    data: collections.pages
-    size: 1
-    alias: pageItem
-permalink: "/{{ '/' if pageItem.slug === 'home' else pageItem.slug  }}/"
```
Comment (greatislander) [inline src/page.njk:6]: I noticed there are whitespace-only changes here. Is there a reason for these?

File: src/news.njk [hunk 2]
```diff
@@ -18,7 +18,7 @@ pagination:
 			{% endfor %}
 		</div>
 		{% if pagination.pages.length > 1 %}
-		  {% include 'components/pagination.njk' %}
+			{% include 'components/pagination.njk' %}
```
Comment (TeddTech) [inline src/news.njk:21]: What was changed in this line?

File: src/news.njk [hunk 3]
```diff
@@ -18,7 +18,7 @@ pagination:
 			{% endfor %}
 		</div>
 		{% if pagination.pages.length > 1 %}
-		  {% include 'components/pagination.njk' %}
```
Comment (greatislander) [inline src/news.njk:21]: I notice a whitespace-only change here. Is there a reason for this?

File: src/page.njk [hunk 4]
```diff
@@ -14,13 +14,13 @@ permalink: "/{{ '/' if pageItem.slug === 'home' else pageItem.slug  }}/"
 {% block content %}
 {% if page.url === '/' %}
 <div>
-    <article>
-    <h1 class="title">{{ pageTitle }}</h1>
-    <div class="api-content">
-    {{ pageItem.content | safe }}
-    </div>
-    </article>
-    <div class="homepage-cards">
```
Comment (greatislander) [inline src/page.njk:23]: I noticed there are whitespace-only changes here. Is there a reason for these?

File: src/page.njk [hunk 5]
```diff
@@ -36,12 +36,28 @@ permalink: "/{{ '/' if pageItem.slug === 'home' else pageItem.slug  }}/"
...
+		{% endfor %}
+	</div>
+</article>
+
 {% else %}
 <article class="page">
-    <h1>{{ pageTitle }}</h1>
-    <div class="api-content">
-        {{ pageItem.content | safe }}
-    </div>
```
Comment (greatislander) [inline src/page.njk:44]: I noticed there are whitespace-only changes here. Is there a reason for these?

**Review by greatislander — CHANGES_REQUESTED** [review]
Noticed some whitespace-only changes; not sure if they were intentional?

File: src/news.njk (hunk 2 above)
Comment (cindyli) [reply src/news.njk:21]: The change here is to replace spaces with a tab.

**Comment by cindyli** [comment]
In answer to all white-space questions, whenever I encounter the use of spaces for indents, I replace them with tabs to comply with our convention of using tabs rather than spaces for indents.

**Review by greatislander — APPROVED** [review]

**Review by TeddTech — COMMENTED** [review]
On line 203 `@include breakpoint-up(sm) {` is the UIO break point. I believe I wrote this in an earlier PR.

**Comment by cindyli** [comment]
> On line 203 `@include breakpoint-up(sm) {` is the UIO break point. I believe I wrote this in an earlier PR.

Fixed the comment in the dev branch @ https://github.com/inclusive-design/wecount.inclusivedesign.ca/commit/8b5ada9f96ee14399e527a02a27f2ea559f21986

**Pull request merged**